docs/modules/ROOT/pages/hazelcast-platform-operator-tls.adoc

```diff
@@ -142,6 +154,28 @@ include::ROOT:example$/python/example.py[]
 +
 [tabs]
 ====
+
+CLC::
++
+--
+Run the following command to fill a map.
+
+[source, shell]
+
+----
+for i in {1..10};
+do
+   clc -c hz map set --name map key-$i value-$i;
+done
+----
```

Comment:
We don't need to fill the map, we just want to show TLS connection is working with CLC. So it's enough to check default map size to see the OK message.